index.html

```diff
@@ -4,35 +4,59 @@
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>Document</title>
+        <link
+            href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+            rel="stylesheet"
+        />
         <link rel="stylesheet" href="./css/index.css" />
     </head>
     <body>
         <header>
+            <div class="hamburger" onclick="toggleMenu()">&#9776;</div>
+            <input type="checkbox" id="menu-toggle" />
+            <!-- navigation -->
             <nav>
-                <ul>
-                    <li><a href="#Skills">Skills</a></li>
-                    <li><a href="#Projects">Projects</a></li>
-                    <li><a href="#About">About</a></li>
-                    <li><a href="#Experience">Experience</a></li>
-                    <li><a href="#Connect">Connect</a></li>
+                <ul id="nav-list">
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#experience">Experience</a></li>
+                    <li><a href="#skills">Skills</a></li>
+                    <li><a href="#projects">Projects</a></li>
+                    <li><a href="#leaveMessage">Leave a message</a></li>
+                    <li><a href="#connect">Connect</a></li>
                 </ul>
             </nav>
+            <!--switch theme-->
+            <div class="theme-switch-wrapper">
+                <label class="theme-switch" for="checkbox">
+                    <input type="checkbox" id="checkbox" />
+                    <div class="slider round"></div>
+                </label>
+            </div>
         </header>
         <main>
             <h1 id="name">Anna Bazileeva</h1>
-
-            <section id="projects">
-                <h2>Projects</h2>
-                <ul></ul>
-            </section>
+            <!-- about -->
             <section id="about">
                 <h2>About</h2>
                 <p>
-                    I don't finish a previous ganemede course. But I hope to
-                    finish haumea class and go forward.
+                    I am a student at
+                    <a href="https://codethedream.org/">Codethedream</a>. I
+                    haven't finish a previous Ganemede course. But I hope to
+                    finish Haumea class and go forward to react course.
                 </p>
-                <img src="./img/IMG_B9F461A8A800-1.jpeg" width="100px" />
+                <img
+                    src="./img/IMG_B9F461A8A800-1.jpeg"
+                    width="100px"
+                    alt="picture"
```

Comment:
This attribute relates to your website accessibility because it is used by screen readers. Try describing what's in the picture.